src/FileHelper.php

```diff
@@ -67,7 +67,7 @@ public static function openFile(string $filename, string $mode, bool $useInclude
      *
      * - It does not fail if directory already exists.
      * - It uses {@see chmod()} to set the permission of the created directory in order to avoid the impact
-     *   of the `umask` setting.
+     *   of the `unmask` setting.
```

Comment:
```suggestion
     *   of the `umask` setting.
```

See https://www.php.net/manual/en/function.mkdir.php